onnxruntime/python/tools/transformers/models/llama/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.  All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+import os
+import sys
+
+sys.path.append(os.path.dirname(__file__))
```

Comment:
Please update the file like 
https://github.com/microsoft/onnxruntime/blob/f044b14e73a7adddc5ebadeaace75f434db8a6a8/onnxruntime/python/tools/transformers/models/whisper/__init__.py

Then you do not need update sys.path in other .py files in this directory.